docs/docs/orgs/README.md

```diff
@@ -1,5 +1,13 @@
 # Managing your Organization
 
+::: info
```

Comment:
I believe you want `::: tip` here:

<img width="715" alt="Screen Shot 2023-01-18 at 12 00 12 PM" src="https://user-images.githubusercontent.com/242668/213282318-04e10d81-e918-4966-83bb-71a18fc83386.png">